DMFT/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.data.mongodb.host=<hostname>
```

Comment:
How are you pulling in the actual values for this environment config?

Reply:
The values in this file would need to be setup based on an instructions document. The values are determined after setting up the DB and adding the required users to it. Once the values are copied to the file (as of now, manually by whoever wants to run the app, but can be automated through devOps); Spring Boot gets all the environment variables from this application.properties file.